### PR TITLE
Time elapsed example

### PR DESCRIPTION
**Pull request by webbywebbers**
This is a little time elapsed example! It uses the image example code as well.

### CODE REVIEW

**Review by GhostofGoes — APPROVED** [review]
LTGM, I'm surprised start_time isn't already in any of the examples

File: examples/time-elapsed-example.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+https://discordapp.com/developers/applications/<APP ID>/rich-presence/assets
+"""
+
+client_id = "client_id"  # Enter your Application ID here.
+RPC = Presence(client_id=client_id)
+RPC.connect()
+
+
+# Make sure you are using the same name that you used when uploading the image
+start_time=time.time()
```
Comment (LewdNeko) [inline examples/time-elapsed-example.py:15]: Could you please put some comments in to explain the large image & start_time args a bit better?

They're documented but lots of people expect to just learn from examples, so well-commented examples are appreciated.

**Review by LewdNeko — COMMENTED** [review]
Code's fine, see other comment though

File: examples/time-elapsed-example.py (hunk 1 above)
Comment (CoopJax) [reply examples/time-elapsed-example.py:15]: Ok, I'll get to it!

**Comment by CoopJax** [comment]
Are those good enough comments? I feel like I should add more to start_time=time.time()

File: examples/time-elapsed-example.py [hunk 2]
```diff
@@ -12,9 +12,9 @@
 
 
 # Make sure you are using the same name that you used when uploading the image
-start_time=time.time()
+start_time=time.time() # Using the time that we imported at the start. start_time equals time.
 RPC.update(large_image="LARGE_IMAGE_HERE", large_text="Programming B)",
```
Comment (LewdNeko) [inline examples/time-elapsed-example.py:16]: I was hoping for a bit of explanation on small and large images but this is fine :)

**Review by LewdNeko — APPROVED** [review]

File: examples/time-elapsed-example.py (hunk 2 above)
Comment (CoopJax) [reply examples/time-elapsed-example.py:16]: There is already a image example so I felt like I didn't need it 😅

**Comment by CoopJax** [comment]
Will this eventually be merged?

**Pull request merged**